webapp/channels/src/actions/views/rhs.ts

```diff
@@ -604,6 +613,7 @@ export function openRHSSearch(): ActionFunc {
     return (dispatch) => {
         dispatch(clearSearch());
         dispatch(updateSearchTerms(''));
+        dispatch(updateSearchTeam(null));
```

Comment:
nit: why using `null` here instead of `''` as we're doing in the rest of the calls/assignations?

Reply:
Because '' is what we use to for "all teams" in other places of the code